spra-play-server/src/main/scala/net/wiringbits/spra/admin/models/FieldValue.scala

```diff
@@ -0,0 +1,6 @@
+package net.wiringbits.spra.admin.models
+
+trait FieldValue extends Serializable
+
+case class StringValue(value: String) extends FieldValue
+case class ByteArrayValue(value: Array[Byte]) extends FieldValue
```

Comment:
I think that this works

```suggestion
sealed trait FieldValue[T] extends Serializable {
  val value: T
}

case class StringValue(value: String) extends FieldValue[String]
case class ByteArrayValue(value: Array[Byte]) extends FieldValue[Array[Byte]]
```